sql/moz-fx-data-shared-prod/org_mozilla_fenix_derived/releases_v1/bigeye_custom_rules.sql

```diff
@@ -0,0 +1,31 @@
+-- {
+--   "name": "Fenix releases version format",
+--   "alert_conditions": "value",
+--   "range": {
+--     "min": 0,
+--     "max": 1
+--   },
+--   "collections": ["Test"],
```

Comment:
I wonder if maybe the 3 keys (collections, owner, and schedule) could just be pulled from the metadata instead to reduce this comment?